gitissues.widget/index.jsx

```diff
@@ -134,11 +140,13 @@ export const updateState = (event, previousState) => {
         'user': issue.user.login,
         'comments': issue.comments,
         'labels': issue.labels.map((l) => { return { 'name': l.name, 'color': l.color } }),
-        'time': t
+        'time': t,
+        'elapsed': elapsed
       })
     }
   }
 
+  previousState.displayIssues = previousState.displayIssues.sort((e1, e2) => (e1.elapsed > e2.elapsed) ? 1 : (e1.elapsed < e2.elapsed) ? -1 : 0)
```

Comment:
```suggestion
  previousState.displayIssues = previousState.displayIssues.sort((e1, e2) => e1.elapsed - e2.elapsed)
```

This can be written simpler